sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_base_prompty_eval.py

```diff
@@ -34,14 +35,17 @@ class PromptyEvaluatorBase(EvaluatorBase[float]):
     :param ignore_queries: If True, queries will be ignored in conversation evaluations. Default is False.
         Useful since some evaluators of this format are response-only.
     :type ignore_queries: bool
+    :keyword passing_score: The minimum score required to pass the evaluation. Defaults to 3.0.
+    :paramtype passing_score: float
     """
 
     LLM_CALL_TIMEOUT = 600
     DEFAULT_OPEN_API_VERSION = "2024-02-15-preview"
 
-    def __init__(self, *, result_key: str, prompty_file: str, model_config: dict, eval_last_turn: bool = False):
+    def __init__(self, *, result_key: str, prompty_file: str, model_config: dict, eval_last_turn: bool = False, passing_score: float):
```

Comment:
Should `passing_score` have a default set? Something like `passing_score: float = DEFAULT_PASSING_SCORE`?

Reply:
There is a default, but I had it being set a few lines below. I can move it to the signature to be clearer.